client/my-sites/earn/home.tsx

```diff
@@ -153,20 +158,36 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 	 * @returns {object} Object with props to render a PromoCard.
 	 */
 	const getReferralsCard = () => {
+		const isJetpackNotAtomic: boolean = isJetpack && ! isAtomicSite;
```

Comment:
This is a real nitpick, but I don't think we need the type there as boolean would be inferred. No harm in leaving it though

Reply:
Removed